lms/static/js/fixtures/search/course_search_form.html

```diff
@@ -2,9 +2,7 @@
     <form>
         <label for="course-search-input" class="sr">Course Search</label>
         <input id="course-search-input" type="text" class="search-field"/>
-        <button type="submit" class="search-button">
-          search <span class="icon fa fa-search" aria-hidden="true"></span>
-        </button>
+        <button type="submit" class="search-button">search</button>
```

Comment:
I wonder if we should capitalize this, since we do with all other such buttons. See the Search box example here: http://ux.edx.org/components/forms/